src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -128,26 +128,34 @@ void EquationInputArea::InputTextBox_Submitted(Object ^ sender, EquationSubmissi
 
 void EquationInputArea::FocusEquationTextBox(EquationViewModel ^ equation)
```

Comment:
For some reason when clicking around in the right pane, an equation text box will suddenly get focus instead of the variable textbox I clicked into. Not sure if related to this PR.